packages/components/psammead-media-player/src/Guidance/index.jsx

```diff
@@ -39,7 +39,7 @@ const GuidanceMessage = styled.strong`
 
 const Guidance = ({ guidanceMessage, service }) => (
   <GuidanceWrapper service={service}>
-    <GuidanceMessage>{guidanceMessage}</GuidanceMessage>
+    <GuidanceMessage aria-hidden="true">{guidanceMessage}</GuidanceMessage>
```

Comment:
This is a boolean prop but we are passing a `string`. It should either be `<GuidanceMessage aria-hidden>` or `<GuidanceMessage aria-hidden={true}>`
```suggestion
    <GuidanceMessage aria-hidden>{guidanceMessage}</GuidanceMessage>
```

Reply:
Hi @sadickisaac, aria-hidden can be `true`, `false` or `undefined`.
- `<GuidanceMessage aria-hidden>` is the same as `<GuidanceMessage aria-hidden="true">`
- `<GuidanceMessage aria-hidden="false">` is not the same as `<GuidanceMessage>`

In this case, your suggestion would save us a few bytes, but I like the verbosity of `aria-hidden="true"` over `aria-hidden` alone. 😄